src/cephadm/cephadm.py

```diff
@@ -4857,12 +4857,14 @@ def _get_parser():
     )
 
     parser_pull = subparsers.add_parser(
-        'pull', help='pull the default container image')
+        'pull',
+        help='pull a ceph container image (will pull the default image if --image not provided)',
+        usage='cephadm pull (for default image) | cephadm --image <image-name> pull (for custom ceph image)')
     parser_pull.set_defaults(func=command_pull)
     parser_pull.add_argument(
         '--insecure',
         action='store_true',
-        help=argparse.SUPPRESS,
+        help='Pull with --tls-verify=false, used for insecure regstries'
```

Comment:
The new option help specifically mentions `--tls-verify=false` which AFAIK doesn't work on docker (at least not on the version I was using a couple of days ago). If the option doesn't literally use it I'd take it out. If the option doesn't work on docker, maybe it should be left as supressed.

Reply:
as discussed offline, it does directly use `--tls-verify=false` which doesn't work with Docker, so keeping it suppressed.